element_electrode_localization/coordinate_framework.py

```diff
@@ -92,6 +136,13 @@ def voxel_query(self, x=None, y=None, z=None):
 
 @schema
 class ParentBrainRegion(dj.Lookup):
+    """Hierarchical structure between the brain regions.
+
+    Attributes:
+        BrainRegionAnnotation.BrainRegion (foreign key): BrainRegionAnnotation.BrainRegion primary key.
+        BrainRegionAnnotation.BrainRegion.proj(parent='acronym') (query): fetches brain region acronym
```

Comment:
in practice, proj looks like a new field renamed
```suggestion
        Parent (query): parent brain region acronym from BrainRegion table
```